pwa-devdocs/src/tutorials/intercept-a-target/change-static-assets-directory/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Change static assets directory
+---
+
+When you build a storefront project based on the Venia template, it copies the [`venia-static` directory][] from the `venia-ui` package into your `dist` directory.
+This directory contains brand-specific files such as icons and the website's `manifest.json` file.
+To use a different set of these files, you will need to create a new static assets directory and copy it into `dist` during build time.
+
+[`venia-static` directory]: https://github.com/magento/pwa-studio/tree/develop/packages/venia-ui/venia-static
+
+## Tasks overview
+
+In this tutorial, you will:
+
+1. Create your own static assets directory
+2. Configure Webpack to copy that directory over during the build process
+3. Configure the UPWARD server to look for specific files in that directory
+4. Update the directory locations in the HTML template
+5. Build and verify your changes
+
+## Create new static assets directory
+
+In your storefront project's root directory, run the following command to copy the `venia-static` directory from the `venia-ui` dependency into a new directory called `my-static-assets`:
+
+``` sh
+cp -r node_modules/@magento/venia-ui/venia-static my-static-assets
```

Comment:
Will `node_modules` always exist immediately after scaffold? Might be worth saying something to the effect of "you must first `yarn install`".